md/bpm-api.md

```diff
@@ -481,6 +481,19 @@ Retrieve humanTask objects that match the specified filters.
   `GET`
 * **Data Params**  
   [Standard search parameters](rest-api-overview.md#resource_search) are available.  
+  
+  * o: only fields available in [HumanTaskInstanceSearchDescriptor] (http://documentation.bonitasoft.com/javadoc/api/7.3/org/bonitasoft/engine/bpm/flownode/HumanTaskInstanceSearchDescriptor.html) and in the result are accepted: 
```

Comment:
Since it is a rest api doc. We don't want to reference any Java class. We just want to list available options here